client/signup/p2-step-wrapper/style.scss

```diff
@@ -1,134 +0,0 @@
-@import url( https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;900&display=swap );
-@import "@wordpress/base-styles/breakpoints";
-@import "@wordpress/base-styles/mixins";
-@import "calypso/assets/stylesheets/p2-extends";
```

Comment:
Let's try removing all those p2 extends. Some of them might still be used in login and invites, but I'd debate if that's necessary.